Sources/DeepLearning/Operators.swift

```diff
@@ -226,14 +226,14 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
     }
 
     @inlinable
-    internal func _vjpMaxPooled(
+    internal func _vjpMaxPooled2D(
         kernelSize: (Int, Int, Int, Int),
         strides: (Int, Int, Int, Int),
         padding: Padding
     ) -> (Tensor, (Tensor) -> Tensor) {
         // TODO: Currently this is not higher order differentiable. Redefine in
         // closed form.
-        let value = maxPooled(kernelSize: kernelSize, strides: strides,
+        let value = maxPooled2D(kernelSize: kernelSize, strides: strides,
```

Comment:
Please merge line below (total length is under 100 columns):
```suggestion
        let value = maxPooled2D(kernelSize: kernelSize, strides: strides, padding: padding)
```